Makefile

```diff
@@ -4,6 +4,10 @@
 test:
 	@go test ./... -cover
 
+# Run tests, plus a docker tests with AIO
+test-with-docker:
```

Comment:
i would name it `integration` if i understand them correctly

cc @roman-khimov, @cthulhu-rider

Reply:
test-integration, test-full, can be any of that.

Reply:
* but comments about using `docker` inside is also a good point

Reply:
renamed to `test-full`